include/kernels/clover_sigma_outer_product.cuh

```diff
@@ -58,10 +58,9 @@ namespace quda
 
     result -= conj(result);
 
-    Matrix<Complex, 3> temp;
-    arg.oprod.load(reinterpret_cast<real *>(temp.data), idx, (mu - 1) * mu / 2 + nu, parity);
+    Matrix<Complex, 3> temp = arg.oprod((mu - 1) * mu / 2 + nu, idx, parity);
```

Comment:
Can this `3` become an `nColor`, or is there a reason it shouldn't be because it's an outer product?

Reply:
trying minimize code changes that are beyond scope of this pull